eeg_flow/oddball/oddball.py

```diff
@@ -0,0 +1,334 @@
+from __future__ import annotations  # c.f. PEP 563, PEP 649
+
+from importlib.resources import files
+from typing import TYPE_CHECKING
+
+import numpy as np
+import psychtoolbox as ptb
+from bsl.lsl import StreamInfo, StreamOutlet
+from bsl.triggers import MockTrigger, ParallelPortTrigger
+from psychopy.core import wait
+from psychopy.sound.backend_ptb import SoundPTB
+from psychopy.visual import ShapeStim, Window
+
+from ..utils._checks import check_type, check_value, ensure_int, ensure_path
+from ..utils.logs import logger
+
+if TYPE_CHECKING:
+    from pathlib import Path
+    from typing import Dict, List, Tuple, Union
+
+
+_TRIAL_LIST_MAPPING = {
+    "100": "trialList_1_100_vol-eval.txt",
+    "600": "trialList_2_600_game-eval.txt",
+    "0a": "trialList_3_360_oddball-%s-a.txt",
+    "0b": "trialList_3_360_oddball-%s-b.txt",
+    "a": "trialList_4_1000_oddball-game-a.txt",
+    "b": "trialList_4_1000_oddball-game-b.txt",
+}
+_DURATION_STIM: float = 0.2  # seconds
+_DURATION_ITI: float = 1.0  # seconds
+_DURATION_FLICKERING: float = 0.05  # seconds
+_TRIGGERS: Dict[str, int] = {
+    "standard": 1,
+    "target": 2,
+    "novel": 3,
+}
+_BACKGROUND_COLOR = (0, 0, 0)  # (r, g, b) between -1 and 1
+_CROSS_WIDTH: int = 20  # pixels
+_CROSS_LENGTH: int = 100  # pixels
+_CROSS_COLOR: str = "black"
+_CROSS_FLICKERING_COLOR: str = "white"
```

Comment:
Task variables to define the cross size, the colors, and the flickering duration.